src/lib/mina/local-blockchain.ts

```diff
@@ -419,6 +420,9 @@ async function LocalBlockchain({
     setProofsEnabled(newProofsEnabled: boolean) {
       this.proofsEnabled = newProofsEnabled;
     },
+    resetProofsEnabled() {
+      this.proofsEnabled = originalProofsEnabled;
+    },
```

Comment:
easier to reset `proofsEnabled` to its original value than to figure out what the original value was and set it